spec/descriptions/getGroupMappings.md

```diff
@@ -0,0 +1,3 @@
+If mappings between groups on the identity provider (LDAP, OIDC, SAML) and Instana groups where configured, this will return a list of those mappings.
+
+This can be configured through the [api](#operation/createGroupMapping) or inside Instana at Settings>IDENTITY PROVIDERS>Group Mapping
```

Comment:
Does "or inside Instana" mean "or on Instana graphical user interface"?

if so I suggest 
`...or on Instana graphical user interface at Settings > Authentication > IDENTITY PROVIDERS > Group Mapping.`

Reply:
On the UI, I will use your suggestion.